lib/test.d.ts

```
@@ -0,0 +1,234 @@
+    export type SyncCallback = (...args: unknown[]) => unknown;
+    export type Callback = (...args: unknown[]) => unknown | Promise<unknown>;
+
+    export type ReturnCall = {
+        args: unknown[];
+        receiver: {};
+        returned: unknown;
+    };
+
+    export type ThrowCall = {
+        args: unknown[];
+        receiver: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)